tools/generators/epic-generator/index.ts

```diff
@@ -0,0 +1,191 @@
+import { getProjects, stripIndents, Tree } from '@nrwl/devkit';
+import { readFileSync } from 'fs';
+import { execSync } from 'child_process';
+import { EpicGeneratorSchema } from './schema';
+import { isPackageConverged } from '../../utils';
+
+function validateSchema(schema: EpicGeneratorSchema): Required<EpicGeneratorSchema> {
+  if (schema.repository !== undefined && !schema.repository.match(/[A-z-]+\/[A-z-]+/)) {
+    throw new Error('Must provide a valid repository with the format {owner}/{repositoryName}');
+  }
+
+  if ((schema.title || '').trim() === '') {
```

Comment:
title is required in schema, so this check is not needed
```suggestion
  if (schema.title.trim().length === 0) {
```